Makefile

```diff
@@ -241,17 +241,6 @@ nitro-audit: build-setup build-container
 			'source /root/.cargo/env && \
 			cargo audit -f /nitro_src/Cargo.lock --ignore RUSTSEC-2020-0159 --ignore RUSTSEC-2020-0071'
 
-nitro-about: build-setup build-container
```

Comment:
Why are you removing this?

Reply:
Right, I can just remove it from the script.

Reply:
Let's merge #425 first.